ui/src/containers/Visualisations/CustomBarChart/index.js

```diff
@@ -0,0 +1,25 @@
+import { compose } from 'recompose';
+import PropTypes from 'prop-types';
+import { Map } from 'immutable';
+import { connect } from 'react-redux';
+import { updateModel } from 'ui/redux/modules/models';
+import { setInMetadata } from 'ui/redux/modules/metadata';
+import Editor from './Editor';
+
+/**
+ * @param {immutable.Map} props.model - visualisation model
+ * @param {string} props.orgTimezone
+ */
+const CustomBarChard = compose(
```

Comment:
Just a little typo here.
```suggestion
const CustomBarChart = compose(
```